src/test/java/com/ericsson/ei/queryservice/test/QueryServiceRESTAPITest.java

```diff
@@ -81,7 +79,7 @@
     private QueryAggregatedObjectControllerImpl aggregatedObjectController;
 
     @MockBean
-    private QueryMissedNotificationControllerImpl missedNotificationController;
+    private FailedNotificationControllerImpl failedNotificationController;
```

Comment:
I would actually prefer to have these tests in a separate class, since the controller for failed notifications are separated from the query controllers.

Reply:
need to be discussed in the team...

Reply:
It is out of scope of this story... you need to create a new story

Reply:
I mentioned this in the issue #342 so I'm resolving the comments here regarding the test division.